packages/astrocite-bibtex/src/parser.ts

```diff
@@ -123,14 +125,26 @@ const parseEntry = (node: Entry, macros: Map<string, string>): Data => {
                         .map(parseName),
                 };
                 break;
-            default:
+            }
+            case 'number': {
```

Comment:
I've wrapped each case in braces, so that `const field` can be used in multiple cases.